js/src/dropdown.js

```diff
@@ -267,22 +266,23 @@ class Dropdown extends BaseComponent {
 
   _getPlacement() {
     const parentDropdown = this._element.parentNode
-    let placement = PLACEMENT_BOTTOM
 
-    // Handle dropup
+    if (parentDropdown.classList.contains(CLASS_NAME_DROPRIGHT)) {
+      return PLACEMENT_RIGHT
+    }
+
+    if (parentDropdown.classList.contains(CLASS_NAME_DROPLEFT)) {
+      return PLACEMENT_LEFT
+    }
+
+    // We need to trim the value because custom properties can also include spaces
+    const isEnd = getComputedStyle(this._menu).getPropertyValue('--bs-position').trim() === 'end'
```

Comment:
I wonder if we can achieve a similar result but without using `getComputedStyle`? Like, looking for a class?

Reply:
Nope. The dropdown classes are responsive. `.dropdown-md-right` is aligned to the right above the md breakpoint. Breakpoints can be added/removed/adjusted so there is no other way to link the CSS with js afaict.

This was one of the reasons why the dropdown alignment is such a pita.